README.md

```diff
@@ -19,6 +19,7 @@ or navigate to any of the documents listed below and download it individually.
 2. [Tutorial: CS231n Python Tutorial](content/cs231_tutorial.md)
 3. [Tutorial: Determining Moore's Law with real data in NumPy](content/mooreslaw-tutorial.ipynb)
 4. [Tutorial: Saving and sharing your NumPy arrays](content/save-load-arrays.ipynb)
+5. [Tutorial: X-ray image processing](content/tutorial-x-ray-image-processing.md)
```

Comment:
@melissawm Just noticed that the MNIST tutorial is also 5th in the list fyi

Reply:
Yeah, I was waiting for that one to be merged to fix this, thanks for catching that :)